test/sync.js

```diff
@@ -71,6 +71,22 @@ test('an additional salt can be provided', t => {
   t.true(actual === expected)
 })
 
+test('salt can be a JSON object', t => {
+  const salt = {foo: 'bar'}
```

Comment:
I thought my linter was checking spaces inside the curly braces, but it's not. So until I manage to fix that you're in the clear 😄